src/communication.ts

```diff
@@ -0,0 +1,65 @@
+import { io, Socket } from 'socket.io-client';
+import { randomBytes } from './util/crypto';
+
+export class Communication {
+    socketServer: Socket;
+
+    constructor() {
+        const url = process.env.REACT_APP_COMMUNICATION_URL ?? 'ws://ms.tonomy.staging';
```

Comment:
this should be set in setSettings from the client. we don't use process.env here